app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Person.cs

```diff
@@ -17,13 +17,21 @@ public class Person
         public string Gender { get; set; }
         public bool? DateDeathConfirmed { get; set; }
         public string Incacerated { get; set; }
+        public string HeightInCms { get; set; }
+        public string WeightInKgs { get; set; }
```

Comment:
I think we should keep the unit out of the field name. I would imagine we would agree to a unit to be used. Can we use Height and Weight.

Reply:
ok

Reply:
will add comments to indicate cms and lb